legend-engine-xt-relationalStore-executionPlan-connection-tests/src/test/java/org/finos/legend/engine/plan/execution/stores/relational/connection/test/ExternalIntegration_TestConnectionAcquisitionWithFlowProvider_BigQuery.java

```diff
@@ -14,158 +14,76 @@
 
 package org.finos.legend.engine.plan.execution.stores.relational.connection.test;
 
+import java.sql.Connection;
+import javax.security.auth.Subject;
 import org.eclipse.collections.api.list.MutableList;
```

Comment:
No chnages should be made in this module, see this PR for refactoring big query tests https://github.com/finos/legend-engine/pull/1007/files . We can write something similar from spanner. De couple spanner tests from big query tests